lib/main.coffee

```diff
@@ -5,22 +5,34 @@ module.exports =
       default: ''
       type: 'string'
       description: 'Leave empty to use bundled'
+    lintInlineJavaScript:
+      type: 'boolean'
+      default: false
+      description: 'Lint JavaScript inside `<script>` blocks in HTML or PHP files'
```

Comment:
I think that if you store scopes in `@scopes` and use `atom.config.observe` in `activate` method and this pass it as grammarScopes here, that should do it.
Also, make sure to dispose the `atom.config.observe`'s disposible on deactivate.